libkbfs/backpressure_disk_limiter.go

```diff
@@ -205,8 +205,8 @@ func (bt *backpressureTracker) beforeDiskBlockCachePut(blockResources int64) (
 	return availableResources
 }
 
-func (bt *backpressureTracker) getLimitInfo() (used int64, limit int64) {
-	return bt.used, bt.limit
+func (bt *backpressureTracker) getLimitInfo() (used int64, limit float64) {
+	return bt.used, bt.currLimit()
```

Comment:
@jzila: `currLimit()` is used in fraction calculations (like is `usedFrac()`), so that doesn't seem like a good solution.  I could cast it to `int64` here if you prefer.  I don't have a strong preference but seemed a little better to only lose information at the highest layer.